test/parallel/test-intl.js

```diff
@@ -54,30 +54,33 @@ if (!haveIntl) {
   }
 
   // Check with toLocaleString
-  var localeString = dtf.format(date0);
-  assert.equal(localeString, 'Jan 70');
-
+  {
+    const localeString = dtf.format(date0);
+    assert.strictEqual(localeString, 'Jan 70');
+  }
   // Options to request GMT
-  var optsGMT = {timeZone: GMT};
+  const optsGMT = {timeZone: GMT};
 
   // Test format
-  localeString = date0.toLocaleString(['en'], optsGMT);
-  assert.equal(localeString, '1/1/1970, 12:00:00 AM');
-
+  {
+    const localeString = date0.toLocaleString(['en'], optsGMT);
+    assert.strictEqual(localeString, '1/1/1970, 12:00:00 AM');
+  }
   // number format
-  assert.equal(new Intl.NumberFormat(['en']).format(12345.67890), '12,345.679');
+  const numberFormat = new Intl.NumberFormat(['en']).format(12345.67890);
+  assert.strictEqual(numberFormat, '12,345.679');
 
-  var collOpts = { sensitivity: 'base', ignorePunctuation: true };
-  var coll = new Intl.Collator(['en'], collOpts);
+  const collOpts = { sensitivity: 'base', ignorePunctuation: true };
+  const coll = new Intl.Collator(['en'], collOpts);
 
-  assert.equal(coll.compare('blackbird', 'black-bird'), 0,
+  assert.strictEqual(coll.compare('blackbird', 'black-bird'), 0,
                'ignore punctuation failed');
```

Comment:
Nit: Can you fix the alignment here and on subsequent lines? `'ignore...` should be aligned under `coll.compare(...`.

I personally kind of wish we didn't align stuff like that, but we do generally, so might as well keep it consistent.